docs/src/main/asciidoc/vertx-reference.adoc

```diff
@@ -694,6 +694,15 @@ Uni<String> response = bus.<String>request("address", "hello, how are you?")
         .onItem().transform(Message::body);
 ----
 
+=== Process events on virtual threads
+
+Methods annotated with `@ConsumeEvent` can also be annotated with `@RunOnVirtualThread`.
+In this case, the method is invoked on a virtual thread.
+Each event is invoked on a different virtual thread.
+
+The method must be _blocking_ and your Java runtime must provide support for virtual threads.
```

Comment:
Does it mean that the return type of the method must not be `Uni`/`Multi`/`CompletionStage`, or what exactly?

Reply:
Yes, it must be either `void` or the object you return (the response to the message you received).

Reply:
Maybe it wouldn't hurt to mention it here so that a user does not need to click a link...?